content/docs/command-reference/move.md

```diff
@@ -164,3 +164,48 @@ $ tree
     │   └── foo
     └── data3.dvc
 ```
+
+## Example: Moving ignored files
+
+`dvc move` raises an error if it finds a file with pattern available in
+`.dvcignore` file.
+
+Let's imagine it with following scenerio:
+
+```dvc
+$ mkdir dir1 dir2
+$ echo data1 > dir1/data1
+$ echo data2 > dir2/data2
+```
+
+Create a `.dvcignore` file and insert pattern matching one of the files:
+
+```dvc
+$ echo dir1/* >> .dvcignore
+```
+
+Now lets try to move the files with `dvc move`.
+
+```dvc
+$ dvc move dir2/data2 dir3/data3
+$ dvc move dir1/data1 dir4/data4
+```
+
+The second command that we executed above will raise an error and the transfer
+of that file won't happen. The final structure would look like this:
```

Comment:
Instead of saying that an error will be thrown, just copy the error output please. Probably no need for that whole paragraph, just continue the code block to include the `tree` command.